prowler/providers/aws/services/ec2/lib/security_groups.py

```diff
@@ -85,9 +90,12 @@ def check_security_group(
                             and ingress_rule["IpProtocol"] == protocol
                         ):
                             return True
-                # If no input ports check if all ports are open
+                # If empty input ports check if all ports are open
                 if len(set(ingress_port_range)) == 65536:
                     return True
+                # If None input ports check if any port is open
+                if ports is None:
```

Comment:
Can you explain this a little bit, I tried but I don't get it.

Reply:
If None, any port will be checked. Which means that if there is at least one port open, it returns true. If empty ([]), all the ports must be open to return True.